### PR TITLE
Removing the extern as we do not need it

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

**Review by vrutkovs — CHANGES_REQUESTED** [review]
Needs formatting updated to make rustfmt

Lets also ensure other `extern crate` are removing from other files as well

**Comment by wking** [comment]
[cargo-test][1] failures like:

```
error[E0433]: failed to resolve: could not find `tokio` in `self`
   --> policy-engine/src/graph.rs:107:15
    |
107 |     use self::tokio::runtime::Runtime;
    |               ^^^^^ could not find `tokio` in `self`
```

I'd guess at least some of these need to stay?

[1]: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_cincinnati/208/pull-ci-openshift-cincinnati-master-cargo-test/777#1:build-log.txt%3A607

**Comment by LalatenduMohanty** [comment]
> I'd guess at least some of these need to stay?

Right, working on it.

**Comment by steveej** [comment]
> 107 |     use self::tokio::runtime::Runtime;
>     |               ^^^^^ could not find `tokio` in `self`

It's worth a try just removing the `self` in use statement.

File: graph-builder/src/main.rs
Comment (vrutkovs) [inline graph-builder/src/main.rs]:
```suggestion
use failure::{Error, ResultExt};
```

File: graph-builder/tests/net/quay_io/mod.rs
Comment (vrutkovs) [inline graph-builder/tests/net/quay_io/mod.rs]:
```suggestion
use cincinnati::{Empty, WouldCycle};
```

File: graph-builder/tests/net/quay_io/mod.rs
Comment (vrutkovs) [inline graph-builder/tests/net/quay_io/mod.rs]:
```suggestion
use graph_builder::registry::{self, fetch_releases, read_credentials, Registry, Release};
```

**Review by vrutkovs — CHANGES_REQUESTED** [review]
Join some imports

**Comment by steveej** [comment]
/lgtm
/hold

AFAIU e2e tests are expected to be in a broken state right now, @vrutkovs please confirm by removing the hold.

**Comment by vrutkovs** [comment]
e2e would fail until #211 lands.

I'm fine with merging this without e2e passing as its purely a code change. WDYT?

**Comment by LalatenduMohanty** [comment]
> I'm fine with merging this without e2e passing as its purely a code change. WDYT?

Lets wait for e2e to get fixed as this pr is not required urgently.

**Comment by vrutkovs** [comment]
/retest

e2e tests have been fixed

**Comment by wking** [comment]
[e2e][1]:

```
error: timed out waiting for the condition on deploymentconfigs/cincinnati
2020/02/11 18:27:22 Container test in pod e2e failed, exit code 1, reason Error
```

/retest

[1]: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_cincinnati/208/pull-ci-openshift-cincinnati-master-e2e/110

**Comment by vrutkovs** [comment]
graph-builder / policy-engine fail to start:
```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /rustc/5e1a799842ba6ed4a57e91f7ab9435947482f7d8/src/libcore/macros/mod.rs:15:40
```
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cincinnati/208/pull-ci-openshift-cincinnati-master-e2e/111/artifacts/e2e/pods/cincinnati-e2e_cincinnati-1-jxqpv_cincinnati-graph-builder.log
That's odd, cause it worked perfectly in previous PR

**Comment by vrutkovs** [comment]
/retest

e2e issue has been fixed

**Pull request merged**
